.env.example

```
@@ -1,14 +1,36 @@
+# ==================== Logging ====================
+LOG_LEVEL=INFO
 
-# Qdrant Cloud Credentials
-# Obtain from: https://cloud.qdrant.io
-QDRANT_API_KEY=your_qdrant_api_key_here
-QDRANT_URL=https://your-cluster.qdrant.io
+# ==================== Spotify API ====================
+SPOTIFY_CLIENT_ID=your_spotify_client_id
+SPOTIFY_CLIENT_SECRET=your_spotify_client_secret
+SPOTIFY_REFRESH_TOKEN=your_spotify_refresh_token
+SPOTIFY_REDIRECT_URI=http://localhost:8888/callback
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SPOTIFY_REDIRECT_URI key should go before the SPOTIFY_REFRESH_TOKEN key

(UnorderedKey)